src/psyclone/parse.py

```diff
@@ -44,7 +44,6 @@ def get_builtin_defs(api):
         fname = None
     return builtins, fname
 
-
```

Comment:
This blank line is required for pep8 compliance.

Reply:
Returned the accidentally deleted blank line so it passes pep8.

Reply:
Corrected BSD license as agreed.